modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -475,6 +476,7 @@ const Main = ({
 	options = [],
 	predefinedValue = [],
 	readOnly = false,
+	defaultSearch = false,
```

Comment:
Hi, @p-albuquerque. Just a simple adjustment. Please, could you sort the `defaultSearch` this property? It needs to be before the `editingLanguageId` prop.
Thanks! =)

Reply:
done.

Reply:
Thanks!! From FE side it's good!